test/jdk/java/awt/print/PrinterJob/DrawImage.java

```diff
@@ -21,262 +21,128 @@
-                        result = printImage(g, _pageFormat, _image);
-                    }
-                    return result;
-                }
-            });
+            pj.setPrintable(this::printImage);
             if (pj.printDialog()) {
-                try { pj.print(); }
-                catch (PrinterException e) {
-                    System.out.println(e);
-                }
+                pj.print();
             }
-
-        }
-        catch (Exception e) {
-            e.printStackTrace(System.out);
-        }
     }
 
-    public static void main(String[] args) {
-                                String[] instructions =
-           {
-            "You must have a printer available to perform this test.",
-            "The test passes if you get a printout of a gray rectangle",
-                                                "with white text without any exception."
-          };
+    private static final String INSTRUCTIONS =
+            "You must have a printer available to perform this test.\n" +
+            "\n" +
+            "The test passes if you get a printout of a gray rectangle\n" +
+            "with white text without any exception.";
 
-         Sysout.createDialog( );
-         Sysout.printInstructions( instructions );
+    public static void main(String[] args) throws Exception {
+
+        if (PrinterJob.lookupPrintServices().length == 0) {
+            throw new SkippedException("Printer not configured or available.");
+        }
 
         BufferedImage image = prepareFrontImage();
+        if (image == null) {
+            throw new RuntimeException("Image creation failed");
+        }
+
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
+                .instructions(INSTRUCTIONS)
+                .rows((int) INSTRUCTIONS.lines().count() + 1)
+                .columns(45)
+                .build();
+
         DrawImage pt = new DrawImage(image);
         pt.print();
-        //      System.exit(0);
+        passFailJFrame.awaitAndCheck();
     }
 
-
-
     public static BufferedImage prepareFrontImage() {
         // build my own test images
         BufferedImage result = new BufferedImage(400, 200,
-                                                 BufferedImage.TYPE_BYTE_GRAY);
+                BufferedImage.TYPE_BYTE_GRAY);
 
-        Graphics2D g2D = (Graphics2D)result.getGraphics();
+        Graphics2D g2D = (Graphics2D) result.getGraphics();
         g2D.setRenderingHint(RenderingHints.KEY_ANTIALIASING,
-                             RenderingHints.VALUE_ANTIALIAS_OFF);
+                RenderingHints.VALUE_ANTIALIAS_OFF);
         int w = result.getWidth(), h = result.getHeight();
 
         g2D.setColor(Color.gray);
         g2D.fill(new Rectangle(0, 0, w, h));
 
         g2D.setColor(Color.white);
 
-        AffineTransform original = g2D.getTransform();
         AffineTransform originXform = AffineTransform.getTranslateInstance(w /
-5, h / 5);
+                5.0, h / 5.0);
```

Comment:
For clarity, wrap all the parameters:
```java
        AffineTransform originXform = AffineTransform.getTranslateInstance(
                w / 5.0, h / 5.0);
```

By the way, this is the case where aligning parameters to start of expression, opening parenthesis doesn't do any good — follow the usual double-intent (8 spaces instead of 4) for the continuation line.